src/components/routes/AccountSettings.tsx

```diff
@@ -431,6 +442,11 @@ export default function AccountSettingsPage() {
 
 	return (
 		<Page>
+			<Snackbar autoHideDuration={6000} open={snackOpen} onClose={closeSnack}>
+				<Alert severity="error" onClose={closeSnack}>
+					Please complete your profile before using chat!
```

Comment:
```suggestion
					Please complete your profile first!
```

If the user clicks "Discord", the current message doesn't make too much sense. This just makes it a bit less specific.